mono/mini/debugger-agent.c

```diff
@@ -2460,6 +2460,12 @@ buffer_add_ptr_id (Buffer *buf, MonoDomain *domain, IdType type, gpointer val)
 	return id;
 }
 
+inline static void
+buffer_add_ptr_id_unsafe(void* buf, void* domain, int type, gpointer val)
+{
+	buffer_add_ptr_id(buf, domain, type, val);
```

Comment:
Will approve, but we should sanity check with Burst team that they aren't relying on return value for this call.

Reply:
@LeeHammerton would you happen to know?

Reply:
> check with Burst team that they aren't relying on return value for this call

I found the header in burst and the signature is void.  The change matches.

Reply:
ah, sorry I missed this, but yeah, it looks fine to me.